app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -189,14 +189,14 @@
           </h3>
         {{/if}}
       </div>
-      <button type="button" class="ui blue small button" {{action 'addTicket' 'free' data.event.tickets.length}}>
+      <button type="button" class="ui blue {{if device.isMobile 'field fluid'}} small button" {{action 'addTicket' 'free' data.event.tickets.length}}>
```

Comment:
Why is the block inserted in the middle of class? Move it to the end

Reply:
@iamareebjamal Done, Moved the block to the end. Please review.